package.xml

```diff
@@ -42,6 +42,7 @@
                     <file name="configuration_php_iface.c" role="src" />
                     <file name="coms.c" role="src" />
                     <file name="coms.h" role="src" />
+                    <file name="curl_handlers.h" role="src" />
```

Comment:
I don't want to 🚲shed too much on naming. I do have one thought though. :)

Can we name this in such a way that all extensions that we provide custom handlers for are grouped together alphabetically? For example, if we named this `handlers_curl`, when we add PDO handlers, it would be `handlers_pdo` and be positioned directly under the curl handlers. What do you think? :)

Reply:
Sure, reasonable enough.